app/tests/spec/models/auth_brokers/fx-ios-v1.js

```diff
@@ -126,23 +130,135 @@ define(function (require, exports, module) {
       });
 
       describe('afterCompleteSignInWithCode', () => {
-        let account;
-
         beforeEach(() => {
           sandbox.spy(broker, 'afterCompleteSignInWithCode');
           sandbox.spy(broker, '_notifyRelierOfLogin');
-          sandbox.spy(FxDesktopV1AuthenticationBroker.prototype, 'afterCompleteSignInWithCode');
-          account = new Account({
-            uid: 'uid'
-          });
+          sandbox.spy(FxiOSAuthenticationBroker.prototype, 'afterCompleteSignInWithCode');
+
           return broker.afterCompleteSignInWithCode(account);
         });
 
         it('broker calls correct methods', () => {
-          assert.isTrue(FxDesktopV1AuthenticationBroker.prototype.afterCompleteSignInWithCode.called);
+          assert.isTrue(broker.afterCompleteSignInWithCode.called);
           assert.isTrue(broker._notifyRelierOfLogin.called);
         });
       });
     });
+
+    describe('createChannel', function () {
```

Comment:
I know the old tests used `function () {`, can you change all of those in these new tests to be `() => {`?